packages/cli/src/options/beaconNodeOptions/chain.ts

```diff
@@ -17,6 +17,7 @@ export interface IChainArgs {
   "chain.assertCorrectProgressiveBalances": boolean;
   "chain.maxSkipSlots": number;
   "safe-slots-to-import-optimistically": number;
+  "chain.archiveStateEpochFrequency": number;
```

Comment:
could be grouped with other `chain.` args

Reply:
yeah, debated doing that. Then settled with "add at the bottom"